README.md

```diff
@@ -24,64 +35,46 @@ Your Jewelbot can also use Bluetooth to detect when another Jewelbot is nearby i
 
 ## Pairing
 
-At the beginning of today's session, the mentors will demonstrate how to pair two Jewelbots. After the demo, try it yourself:
+At the beginning of today's session, we'll demonstrate how to pair two Jewelbots. After the demo, try it yourself:
 
 1. Find another girl to pair up with.
 2. Unplug both of your Jewelbots and make sure they're both turned on.
 3. Press the magic button on one Jewelbot for 2 seconds to put it into pairing mode. It should slowly flash all white lights.
-4. Press the magic button on the other Jewelbot for 2 seconds to put it into pairing mode also. The first Jewelbot should now stay all white while the second one cycles through different colors.
-5. Choose a friendship color by pressing the magic button on the second Jewelbot when it's showing that color.
+4. Press the magic button on the other Jewelbot for 2 seconds to put it into pairing mode also. One Jewelbot should now stay all white while the other cycles through different colors.
```

Comment:
"2 seconds" -> "2 seconds(2 Mississippi)"

Reply:
I didn't change this because it's already there in an earlier part of the worksheet.